libraries/Microsoft.Bot.Schema/Teams/TargetedMeetingNotificationValue.cs

```diff
@@ -7,42 +7,40 @@
 namespace Microsoft.Bot.Schema.Teams
 {
     /// <summary>
-    /// Specifies the container for what is required to send a meeting notification to recipients.
+    /// Specifies the targeted meeting notification value, including recipients and surfaces.
     /// </summary>
-    public partial class TeamsMeetingNotificationInfo
+    public partial class TargetedMeetingNotificationValue
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="TeamsMeetingNotificationInfo"/> class.
+        /// Initializes a new instance of the <see cref="TargetedMeetingNotificationValue"/> class.
         /// </summary>
-        public TeamsMeetingNotificationInfo()
+        public TargetedMeetingNotificationValue()
         {
             CustomInit();
         }
 
         /// <summary>
-        /// Gets or sets the collection of recipients of the notification.
+        /// Gets or sets the collection of recipients of the targeted meeting notification.
         /// </summary>
         /// <value>
-        /// The collection of recipients of the notification.
+        /// The collection of recipients of the targeted meeting notification.
         /// </value>
         [JsonProperty(PropertyName = "recipients")]
 #pragma warning disable CA2227 // Collection properties should be read only (we can't change this without breaking binary compat)
         public IList<string> Recipients { get; set; }
 
         /// <summary>
         /// Gets or sets the collection of surfaces on which to show the notification.
+        /// If a bot wants its content to be rendered in multiple surfaces areas, it can specific a list of UX areas. 
```

Comment:
Nit: how about we use '_different_ surface areas' instead of '_multiple_ surface areas'?

Reply:
Could you share why you think different is better than multiple here? Is that because "only one instance of surface type is allowed per request"?
This originally came from Dev doc and it makes me a little confused.

Reply:
Cool. Will update it in the next iteration.